README.md

```diff
@@ -37,4 +37,10 @@ Development
 Use Existing Network
 --------------------
 
-To use existing network at NIOS, The network should be assigned with EA`s
\ No newline at end of file
+By default the network is created using network configuration(infoblox-ipam.conf), When you want to use existing networks at the NIOS just update EA`s(Network Name,CMP Typ,Cloud API Owned,Tenant ID) to the pre existing networks.
```

Comment:
By default required network view & network will be created based on configuration available at infoblox-ipam.conf. if you want to use existing network view & networks available at the NIOS update below Extensible Attribute (EA).